test/parallel/test-event-target.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const eventPhases = {
+  'NONE': 0,
+  'CAPTURING_PHASE': 1,
+  'AT_TARGET': 2,
+  'BUBBLING_PHASE': 3
+};
+
+// Using Object.keys to get the properties from the Event object
+const eventKeys = Object.keys(Event);
+
+for (const [prop, value] of Object.entries(eventPhases)) {
+  // Check if the property exists in the Event object
+  assert.ok(eventKeys.includes(prop), `Event does not have the property: ${prop}`);
```

Comment:
Is this check really required? It will iterate through all the keys for each phase. 

Anyway, we are accessing a property from the below lines, so if it's not exist errors will be thrown.

Or else, if you really want to have this check, you may use a `truthy` check for  `Event[prop]`. So it will ensure less complexity on each run and avoid includes. 

I mean check whether the `Event[prop`] is defined or not instead of iteration.

Reply:
Yea, it might not be required. Can remove it if required, I just thought since we are checking the value later, might as well check it's existence, but like you said, it will trigger the other check anyways.